jkube-kit/build/service/jib/src/main/java/org/eclipse/jkube/kit/service/jib/JibServiceUtil.java

```diff
@@ -118,7 +118,7 @@ public static void buildContainer(JibContainerBuilder jibContainerBuilder, TarIm
     public static JibContainerBuilder containerFromImageConfiguration(
         ImageConfiguration imageConfiguration, Credential pullRegistryCredential) throws InvalidImageReferenceException {
         final JibContainerBuilder containerBuilder = Jib.from(getRegistryImage(getBaseImage(imageConfiguration), pullRegistryCredential))
-                .setFormat(ImageFormat.OCI);
+                .setFormat(ImageFormat.Docker);
```

Comment:
What's the purpose of this change?

Reply:
Not sure why it was working in previous versions. If I set image format to `ImageFormat.OCI` then the generated tarball can't be loaded by docker when using `docker load -i` step.

Reply:
:+1: 
Maybe they updated the OCI spec version or added incompatible entries.